src/braket/default_simulator/gate_operations.py

```diff
@@ -230,7 +230,7 @@ def matrix(self) -> np.ndarray:
             1
             / np.sqrt(2)
             * np.array(
-                [[0, 1, 0, 1.0j], [1, 0, -1.0j, 0], [0, 1.0j, 0, 1], [-1.0j, 0, 1, 0]],
+                [[0, 0, 1, 1.0j], [0, 0, 1.0j, 1], [1, -1.0j, 0, 0], [-1.0j, 1, 0, 0]],
```

Comment:
This change to reflect the [bdk implementation](https://github.com/aws/amazon-braket-sdk-python/blob/main/src/braket/circuits/gates.py#L1324)